modules/cluster-logging-uninstall.adoc

```diff
@@ -60,9 +60,33 @@ Do not delete the `openshift-operators-redhat` project if other global operators
 
 .. Confirm the deletion by typing `openshift-operators-redhat` in the dialog box and click *Delete*.
 
-. Optional: Remove any OpenShift Logging persistent volume claims (PVC):
+.Next steps
+
+Uninstalling OpenShift Logging does not remove the persistent volume claims (PVCs). To preserve or delete the PVCs, persistent volumes (PVs), and associated data, you must take further action.
+
+
+[WARNING]
+====
+Releasing or deleting PVCs can delete PVs and cause data loss.
+====
+
+. Evaluate whether to keep the PVCs for reuse with other pods. If so, keep the labels or PVC names you need to reclaim the PVCs.
```

Comment:
This seems like two steps in one. You could shorten this to a concise single step that leads into the second step. For example:

". If you want to keep the PVCs for reuse with other pods, you must keep the labels or PVC names that you need to reclaim the PVCs."

You could also change "names you need" to "names that you need", as seen in my example, to remove potential ambiguity.